pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -85,6 +86,18 @@ export default Vue.extend<Data, Methods, Computed, never>({
 
       return date.format('YYYY-MM-DD HH:mm');
     },
+    escapeSnapshotName(value: string|undefined) {
```

Comment:
We already have an [`escapeHtml`](https://github.com/rancher-sandbox/rancher-desktop/blob/0efb3bd9f0493b5060a73f6d790da82cd0685452/pkg/rancher-desktop/utils/string.js#L57) that's used by the [i18n code](https://github.com/rancher-sandbox/rancher-desktop/blob/0efb3bd9f0493b5060a73f6d790da82cd0685452/pkg/rancher-desktop/plugins/i18n.js#L24); perhaps we can re-use it?

Reply:
reused-it